client/client_test.go

```diff
@@ -5538,6 +5540,7 @@ func testRelativeMountpoint(t *testing.T, sb integration.Sandbox) {
 
 // moby/buildkit#2476
 func testBuildInfoExporter(t *testing.T, sb integration.Sandbox) {
+	integration.SkipIfDockerd(t, sb, "image export")
```

Comment:
Image exporter is supported(but it might need switch because I think it is called "moby"). Direct push is not supported. That should be the reason for the current test.